pipeline-templates/job/code-build.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  SolutionBaseName:
+  BuildConfiguration:
+
+jobs:
+- job: CodeBuild
+  variables:
+  - group: BUILD Management Resources

```

Comment:
Casing of `BUILD` is inconsistent

Reply:
Simplified

Reply:
The new variable group 'BUILD Management Resources' matches the convention of new variable groups for capital initial 'environment'.

I think PascalCase for job name might be OK for readability

Reply:
Agreed